introduction.md

```diff
@@ -2,4 +2,9 @@ Hello! I am testing out a course at codeschool.com.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?